Jellyfin.Api/Constants/UserRole.cs

```diff
@@ -0,0 +1,23 @@
+namespace Jellyfin.Api.Constants
+{
+    /// <summary>
+    /// Constants for user roles used in the authentication and authorization for the API.
+    /// </summary>
+    public static class UserRole
```

Comment:
```suggestion
    public static class UserRoles
```
Since this isn't a enum anymore